web-console/4.0-designs/storage/Expand Storage Cluster/Expand Storage Cluster.md

```diff
@@ -0,0 +1,73 @@
+# Expand OCS Cluster
+
+## Supported Deployments for OCS 4.2
+
+- AWS (IPI & UPI)
+- VMware (UPI only)
+- RHHI.Next (Bare metal)
+
+## Expand Resource Wireframes 
+We added to the storage section a new entity for “Storage Resources” which lists all the different storage platforms that are applicable or currently configured on OpenShift:
+
+## Add Capacity
+To add capacity, the flow will be decided based on the platform OpenShift is deployed on:
+
+- **Add Capacity** - add capacity by creating underlying storage devices/volumes (in the cloud, i.e. AWS/cloud)
+- **Add Storage Devices (4.3)** - use existing nodes & add available storage devices/volumes to select from (for UPI/unmanaged, i.e. VMware / Baremetal)
+
+### Add Capacity (AWS/cloud)
+![Storage Resources List](img/Expand-Storage-01.png) 
```

Comment:
@yuvalgalanti 
Please add the "MCG" tab after the "OCS" tab in all the mockups after "All Instances, " OCS".